src/installer/corehost/error_codes.h

```diff
@@ -51,6 +51,7 @@ enum StatusCode
     HostInvalidState                    = 0x800080a3,
     HostPropertyNotFound                = 0x800080a4,
     CoreHostIncompatibleConfig          = 0x800080a5,
+    HostApiUnsupportedScenario          = 0x800080a6,
```

Comment:
Can you please add this to https://github.com/dotnet/runtime/blob/master/docs/design/features/host-error-codes.md